README.md

```diff
@@ -114,98 +82,124 @@ there:
 
        ./dependencies/down.sh
 
+### Running
+
 1. Start the Lighthouse service specified in the `docker-compose.yml` from the
    root of the repository:
 
         docker-compose up
 
-1. Create a `.env` file which contains the line:
+1. Find the lighthouse container id with:
 
-        LOCALHOST=host.docker.internal
+        docker ps
 
-1. Start the docker container and open a bash session in it with:
+1. Start a bash session in the container with:
 
-        docker run --env-file .env -p 8000:8000 -v $(pwd):/code -it --entrypoint bash lighthouse:develop
+        docker exec -ti <container_id for lighthouse> bash
 
    After this command you will be inside a bash session inside the container of lighthouse, and will have mounted all
-   source code of the project from your hosting machine The container will map your port 8000 with the port 8000 of
-   Docker.
-
-1. Once inside the docker container, run:
+   source code of the project from your hosting machine. The container will map your port 8000 with the port 8000 of
+   Docker (configured in docker-compose.yml).
 
-        pipenv install
-        pipenv shell
-
-1. Now that you are inside the virtual environment, initialize the development database for SQLServer:
+1. Now that you are inside the virtual environment, initialize the MySQL and SQLServer development databases:
```

Comment:
```suggestion
1. Now that you are inside the running container, initialize the MySQL and SQLServer development databases:
```

Reply:
Not sure either, but I'm going to leave it for now, as my mission was just to update the readme :)